### PR TITLE
Fix "page" and "delete" translation

### PR DESCRIPTION
**Pull request by StalderT**

### CODE REVIEW

File: editor/locale/lang.fa.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ svgEditor.readLang({
 		"copy": "Copy",
 		"paste": "Paste",
 		"paste_in_place": "Paste in Place",
-		"‫حذف ‬": "Delete",
+		"delete ‬": "Delete",
```
Comment (progers) [inline editor/locale/lang.fa.js]: Should this be "delete": "حذف ‬"?

File: editor/locale/lang.ro.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ svgEditor.readLang({
 		"fit_to_sel": "Potrivire la selecţie",
 		"align_relative_to": "Aliniere în raport cu ...",
 		"relativeTo": "în raport cu:",
-		"de start": "de start",
+		"page": "Page",
```
Comment (progers) [inline editor/locale/lang.ro.js]: Should this be "page": "de start"? Or maybe "page": "pagină"?

File: editor/locale/lang.sw.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ svgEditor.readLang({
 		"fit_to_sel": "Waliopo uteuzi",
 		"align_relative_to": "Align jamaa na ...",
 		"relativeTo": "relativa att:",
-		"umebadilisha": "umebadilisha",
+		"page": "Page",
```
Comment (progers) [inline editor/locale/lang.sw.js]: Should this be "page": "umebadilisha"?

File: editor/locale/lang.vi.js [hunk 4]
```diff
@@ -79,7 +79,7 @@ svgEditor.readLang({
 		"fit_to_sel": "Phù hợp để lựa chọn",
 		"align_relative_to": "Căn liên quan đến ...",
 		"relativeTo": "liên quan đến:",
-		"Sửa": "Sửa",
+		"page": "Page",
```
Comment (progers) [inline editor/locale/lang.vi.js]: Should this be "page": "Sửa"?

File: editor/locale/lang.yi.js [hunk 5]
```diff
@@ -79,7 +79,7 @@ svgEditor.readLang({
 		"fit_to_sel": "פּאַסיק צו אָפּקלייב",
 		"align_relative_to": "יינרייען קאָרעוו צו ...",
 		"relativeTo": "קאָרעוו צו:",
-		"בלאַט": "בלאַט",
+		"page": "Page",
```
Comment (progers) [inline editor/locale/lang.yi.js]: Should this be "page": "בלאַט"?

File: editor/locale/lang.ro.js (hunk 2 above)
Comment (StalderT) [inline editor/locale/lang.ro.js]: ok : https://github.com/SVG-Edit/svgedit/commit/e4875ba514c05b3e355770c0aa5c56b5e994ec58

File: editor/locale/lang.sw.js (hunk 3 above)
Comment (StalderT) [inline editor/locale/lang.sw.js]: No idea : google translate say "umebadilisha"="has changed"

File: editor/locale/lang.vi.js (hunk 4 above)
Comment (StalderT) [inline editor/locale/lang.vi.js]: No idea : google translate say "Sửa"="Milk, repair, modify"

File: editor/locale/lang.yi.js (hunk 5 above)
Comment (StalderT) [inline editor/locale/lang.yi.js]: Ok : https://github.com/SVG-Edit/svgedit/commit/693de51a2118468d6b3a6ecc9acb93f949ece7b6

File: editor/locale/lang.fa.js (hunk 1 above)
Comment (StalderT) [inline editor/locale/lang.fa.js]: Ok : https://github.com/SVG-Edit/svgedit/commit/c40c67748fe18d8c2cae246b6b8310142e088eda

File: editor/locale/lang.vi.js (hunk 4 above)
Comment (progers) [inline editor/locale/lang.vi.js]: Ahh, I didn't even check the translation of this one. I thought it was just a mistake. Ok, this seems better than was there before, as we shouldn't be milking any pages in svgedit (here, and the other "no idea" comments on this review). Lets merge this :)

**Pull request merged**